src/setData/dlc1.js

```diff
@@ -442,15 +442,21 @@ module.exports = [
             ],
           },
           {
-            title: 'Achievement Emblems',
+            title: 'Emblems',
+            items: [
+              1312472209, // Fields of Glass
+              2939572589, // Crossroads Emblem
+              4261480748, // Transit of Mercury
+            ],
+          },
+          {
+            title: 'Achievement Emblems - Curse of Osiris',
```

Comment:
Don't think the title needs " - Curse of Osiris" as Mercury is only available with Curse of Osiris and this section is on that page. Apart from that, very good changes. I'll close my PR as my changes have been put in here

Reply:
I was matching what was done on the base game page. I try to be consistent even if it is redundant, but I can see going for a streamlined version so I removed it from the title.